src/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -72,6 +72,17 @@ public ConfidentialClientApplicationBuilder WithClientSecret(string clientSecret
             return this;
         }
 
+        /// <summary>
+        /// Sets the application client assertion
```

Comment:
This comment is not descriptive enough. It should have more details and an aka.ms link 

CC @jmprieur who should review all public comments

Reply:
Should there be a link to a wiki page describing the Client Assertion? (probably already available somewhere)

---
In reply to: [292420638](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1208#discussion_r292420638) [](ancestors = 292420638)

Reply:
Wiki and comments will be updated early next week when @jmprieur returns